Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -1425,6 +1426,25 @@ def get_current_time_in_seconds() -> float:
     return time.time_ns() / (10 ** 9)
 
 
+def create_files_to_upload(file_mime_type: str, file_name: str, file_bytes: bytes, attachment_name: str | None = None) -> dict:
+    """ Creates the file object to upload to Jira
+        Args:
+            file_mime_type (str): The mime type of the file.
+            file_name (str): The name of the file.
+            file_bytes(bytes): The bytes of the file.
+            attachment_name (str | None): A custom attachment name, if it is empty or None then the attachment's name will be the
+            same one as in XSOAR. Default is None
+
+        Returns:
+            [Dict[tuple(str, bytes, str)]]: The file object of new attachment (file name, content in bytes, mime type).
+        """
+    # guess_type can return a None mime type if the type can’t be guessed (missing or unknown suffix). In this case, we should use
+    # a default mime type
+    file_mime_type = file_mime_type if file_mime_type else guess_type(file_name)[0] or 'application-type'
+    demisto.debug(f'In create_files_to_upload {file_mime_type=}')
+    return {'file': (attachment_name or file_name, file_bytes, file_mime_type)}
```

Comment:
To differentiate between the method argument and final MIME type.
```suggestion
    mime_type_to_upload = file_mime_type if file_mime_type else guess_type(file_name)[0] or 'application-type'
    demisto.debug(f'In create_files_to_upload {mime_type_to_upload=}')
    return {'file': (attachment_name or file_name, file_bytes, mime_type_to_upload)}
```